node/src/Transaction.ts

```diff
@@ -1409,6 +1412,57 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         );
     }
 
+    /**
+     * Returns a random element from the sorted set stored at `key`.
+     *
+     * See https://valkey.io/commands/zrandmember/ for more details.
+     *
+     * @param keys - The key of the sorted set.
+     *
+     * Command Response - A string representing a random element from the sorted set.
+     *     If the sorted set does not exist or is empty, the response will be `null`.
+     */
+    public zrandmember(key: string): T {
```

Comment:
```suggestion
    public async zrandmember(key: string): T {
```

Reply:
Adding async here actually breaks the static check and gets a prompt to use `Promise<T>` instead of `T` as return type